portality/forms/application_forms.py

```diff
@@ -547,10 +547,27 @@ class FieldDefinitions:
                           "a society or other type of institution, enter that here."],
             "placeholder": "Type or select the society or institution’s name"
         },
+        "contexts" : {
+            "admin": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                ]
+            },
+            "associate_editor": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                ]
+            },
+            "editor": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                ]
+            }
+        },
```

Comment:
THis overwrites the widgets on the base field, so it needs to include the `autocomplete` and any other widgets that you also want active on the admin form.